rpcs3/rpcs3qt/vfs_dialog_usb_input.cpp

```diff
@@ -0,0 +1,99 @@
+#include "vfs_dialog_usb_input.h"
+#include "gui_settings.h"
+
+#include <QDialogButtonBox>
+#include <QPushButton>
+#include <QVBoxLayout>
+
+#include "Emu/vfs_config.h"
+
+inline std::string sstr(const QString& _in) { return _in.toStdString(); }
+
+vfs_dialog_usb_input::vfs_dialog_usb_input(const QString& name, const cfg::device_info& default_info, cfg::device_info* info, std::shared_ptr<gui_settings> guiSettings, QWidget* parent)
+	: QDialog(parent), m_gui_settings(std::move(guiSettings)), m_gui_save(gui::fs_dev_usb_list)
+{
+	if (!info || name.back() < '0' || name.back() > '7')
+	{
+		fmt::throw_exception("OH BOY, THIS IS BAD");
```

Comment:
Proper exception message?

Reply:
It's a unique string and not supposed to happen unless I really forgot to add something during development.
Also, it's boring otherwise